landing-page/index.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+   <title>CtD class-Exerc.3-Landing page</title>
+   <link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+
+<body>
+  <header id="showcase">
```

Comment:
The `<head>` and `<body>` tags are nested beneath the `<html>` tag, so they should be indented 2 spaces